bigtable/spark/src/test/scala/example/IntegrationTest.scala

```diff
@@ -0,0 +1,43 @@
+package example
+
+import com.google.bigtable.repackaged.com.google.cloud.bigtable.data.v2.models.Query
+import com.google.bigtable.repackaged.com.google.cloud.bigtable.data.v2.{BigtableDataClient, BigtableDataSettings}
+import org.scalatest.flatspec._
+import org.scalatest.matchers._
+
+class IntegrationTest extends AnyFlatSpec
+    with should.Matchers {
+
+  def getOrThrowException(envName: String): String = {
+    sys.env.getOrElse(
+      envName,
+      throw new IllegalStateException(s"Environment variable '$envName' is required to perform this integration test."))
+  }
+  val projectId = getOrThrowException("BIGTABLE_SPARK_PROJECT_ID")
+  val instanceId = getOrThrowException("BIGTABLE_SPARK_INSTANCE_ID")
+  val table_wordcount = getOrThrowException("BIGTABLE_SPARK_WORDCOUNT_TABLE")
+  val file = getOrThrowException("BIGTABLE_SPARK_WORDCOUNT_FILE")
+  val table_copytable = getOrThrowException("BIGTABLE_SPARK_COPYTABLE_TABLE")
+  val rowCount = getOrThrowException("BIGTABLE_SPARK_ROW_COUNT").toInt
+
+  "IntegrationTest" should "write records to Bigtable, copy them between tables" in {
+    import org.apache.spark.{SparkConf, SparkContext}
+    val appName = getClass.getSimpleName.replace("$", "")
+    val config = new SparkConf().setMaster("local[*]").setAppName(appName)
+    SparkContext.getOrCreate(config)
+
+    val wordcountArgs = Array(projectId, instanceId, table_wordcount, file)
+    Wordcount.main(wordcountArgs)
+    val copytableArgs = Array(projectId, instanceId, table_wordcount, table_copytable)
+    CopyTable.main(copytableArgs)
+
+    val settings =
+      BigtableDataSettings.newBuilder().setProjectId(projectId).setInstanceId(instanceId).build()
+    val dataClient = BigtableDataClient.create(settings)
+    import collection.JavaConverters._
+    val wordcountRowCount = dataClient.readRows(Query.create(table_wordcount)).iterator().asScala.length
+    val copytableRowCount = dataClient.readRows(Query.create(table_copytable)).iterator().asScala.length
+    wordcountRowCount should be(rowCount)
+    wordcountRowCount should be(copytableRowCount)
```

Comment:
looks good! Only thing I think needs to be added is a cleanup function to remove the data added to the table after it's finished with the assertions.